### PR TITLE
Frio: fix contact filtering on contacts page

### LINKED ISSUE #4923: frio: search in /contacts doesn't work for username
### Expected behavior
search for username like myname@domain.tld should find the `myusername`
Dosn't work in frio, but works in vier

### Actual behavior
only works with the users first- and lastname.

### Steps to reproduce the problem
Just use the search.

Problem with all that is, that some ☇☈☋♐⚔ people are using symbols instead of characters, then you can't find them via search and you have to scroll all of your contacts.

### tec
Friendica version: 2018-05-dev
Friendica source: git
PHP version: 7.0
SQL version: mariadb

### PR DESCRIPTION
**Pull request by rabuzarus**
Fixes #4923

### CODE REVIEW

**Comment by rabuzarus** [comment]
BTW: Should we change the filtering behavior in the way that it only starts at the begin of a name/nick?
Means, changing 
```AND (`attag` LIKE '%%" . dbesc($search_txt) . "%%' OR `name` LIKE '%%" . dbesc($search_txt) . "%%' OR `nick` LIKE '%%" . dbesc($search_txt) . "%%')```
to
```AND (`attag` LIKE ' " . dbesc($search_txt) . "%%' OR `name` LIKE ' " . dbesc($search_txt) . "%%' OR `nick` LIKE ' " . dbesc($search_txt) . "%%')```

File: view/theme/frio/js/theme.js [hunk 1]
```diff
@@ -550,6 +550,10 @@ function filter_replace(item) {
 		this.attr('autocomplete','off');
 		var a = this.textcomplete([contacts], {className:'accontacts', appendTo: '#contact-list'});
 
+		if(autosubmit) {
+			a.on('textComplete:select', function(e,value,strategy) {submit_form(this);});
```
Comment (MrPetovan) [inline view/theme/frio/js/theme.js]: What is this for?

Comment (rabuzarus) [reply view/theme/frio/js/theme.js]: to use the original search function on enter

**Comment by MrPetovan** [comment]
> Should we change the filtering behavior in the way that it only starts at the begin of a name/nick?

If you do that, then you have to provide a wildcard character to search anywhere in the name nick, which isn't ideal. I'd say no.

**Pull request merged**